c2corg_ui/static/js/lang.js

```diff
@@ -84,6 +84,7 @@ app.LangController.prototype.updateCulture = function() {
   this.gettextCatalog_.setCurrentLanguage(this.culture);
   this.gettextCatalog_.loadRemote(
       this.langUrlTemplate_.replace('__lang__', this.culture));
+  window.localStorage.setItem('interface_lang', this.culture);
```

Comment:
I agree non sensitive info should simply be stored in local storage.
The try catch is for iOS in incognito mode.